tests/subsys/bluetooth/mesh/light_ctrl/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+cmake_minimum_required(VERSION 3.13.1)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(bt_mesh_ligth_ctrl_sybsystem_test)
+
+FILE(GLOB app_sources src/*.c)
+
+target_sources(app 
+  PRIVATE
+  ${app_sources}
+  ${NRF_DIR}/subsys/bluetooth/mesh/light_ctrl_srv.c
+  ${ZEPHYR_BASE}/subsys/net/buf.c
+  )
+
+target_include_directories(app 
+  PRIVATE
+  ${NRF_DIR}/subsys/bluetooth/mesh
+  ${ZEPHYR_BASE}/subsys/bluetooth
+  )
+
+target_compile_options(app
+  PRIVATE
+  -DCONFIG_BT_MESH_MODEL_KEY_COUNT=5
+  -DCONFIG_BT_MESH_MODEL_GROUP_COUNT=5
+  -DCONFIG_BT_LOG_LEVEL=256
+  -DCONFIG_BT_MESH_LIGHT_CTRL_SRV=1
+  -DCONFIG_BT_MESH_LIGHT_CTRL_SRV_RESUME_DELAY=0
+  -DCONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_MANUAL=5
+  -DCONFIG_BT_MESH_LIGHT_CTRL_SRV_OCCUPANCY_MODE=1
+  -DCONFIG_BT_MESH_LIGHT_CTRL_SRV_OCCUPANCY_DELAY=0
+  -DCONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_FADE_ON=500
+  -DCONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_ON=3
+  -DCONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_FADE_PROLONG=5000
+  -DCONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_FADE_STANDBY_AUTO=5000
+  -DCONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_PROLONG=3
+  -DCONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_FADE_STANDBY_MANUAL=500
+  -DCONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_STANDBY=0
+  -DCONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_ON=65535
+  -DCONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_PROLONG=10000
```

Comment:
These should all be placed in prj.conf.

Reply:
I can not put this into the prj.conf file, because then it will complain about dependencies. And then we need to start adding all of mesh. I just want to compile the one file that need these defines, and not the hole mesh and Bluetooth.